### DISCUSSION TITLE
Add Euler-Mascheroni constant to the constants values

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/runtime#72521] [General] [answered]
There is currently E, Pi, Tau in System.Math.
However Euler-Mascheroni constant is no where to be found.

**Comment by Xyncgas** [comment, marked as answer]
https://ehfd.github.io/world-record/euler-mascheroni-constant/

**Reply by bernd5** [reply]
that should be reported in https://github.com/dotnet/runtime

**Reply by Xyncgas** [reply]
Thanks

**Comment by HaloFour** [comment]
That'd be a request for the [runtime repo](https://github.com/dotnet/runtime) to add another constant member to the `System.Math` class.
